server/interfaces/ISocketJsonable.ts

```diff
@@ -0,0 +1,3 @@
+export interface ISocketJsonable {
+    toSocketJson(): unknown;
```

Comment:
type unknow ? c'est normal ? j'imagine

Reply:
Yes, this could be technically set as a type which would be almost exactly the same as a Socket, but I feel like this would be overkill.